docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -112,7 +110,52 @@ To send a command from the client and to process it on the server side:
     }
     ```
 
-<a name="servermetadata"></a>
+## Updating and Retreiving Playlist
+
+To create a playlist from the server side:
+
+1. Create a new instance of the [Tizen.Multimedia.Remoting.MediaControlPlaylist](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Remoting.MediaControlPlaylist.html) class:
+
+    ```
+    var playlist = new MediaControlPlaylist("MyFavorite");
+    ```
+
+You can create playlist with Metadata also like below code.
+
+    ```
+	var playlist = new MediaControlPlaylist("MyFavoriate",
+		new Dictionary<string, MediaControlMetadata>()
+		{
+			{"Song1", new MediaControlMetadata() {Author = "Someone"} },
+			{"Song2", new MediaControlMetadata()}
+		}
+    );
+	```
+
+To retrieve the playlist and metadata information on the client side:
+    ```
+	    controller.PlaylistUpdated += (s, e) =>
+		{
+		    Tizen.Log.Info(LogTag, $"Updated mode : {e.Mode}, Name : {e.Name}, The title of first song : {e.Playlist.GetMetadata("IDX1").Title}");
+		};
+	````
+
+## Setting and Getting Playback Capability
+
+To get playback capability from the client and to set it on the server side:
```

Comment:
According to the Google Style Guide kindly add "follow these steps" to this sentence:

Suggested edit: "To get playback capability from the client and to set it on the server side, follow these steps:"

Reply:
Done